src/main/java/com/vesoft/nebula/jdbc/impl/NebulaDriver.java

```diff
@@ -39,119 +37,80 @@ public class NebulaDriver extends NebulaAbstractDriver {
         log.info("Driver registered");
     }
 
+    @Override
+    public boolean acceptsURL(String url) throws SQLException {
+        if (url == null) {
+            throw new SQLException("Invalid url: null.");
+        }
+        String[] pieces = url.split("//");
+        return url.startsWith(JDBC_NEBULA_PREFIX) && pieces.length == 2;
+    }
 
-    protected Session getSessionFromNebulaPool() throws SQLException {
-
-        String user = (String) connectionConfig.getOrDefault("user", "root");
-        String password = (String) connectionConfig.getOrDefault("password", "nebula");
-        boolean reconnect = (boolean) connectionConfig.getOrDefault("reconnect", false);
+    protected Properties parseUrlProperties(String url, Properties connectionConfig) throws URISyntaxException {
+        return NebulaJdbcUrlParser.parse(url, connectionConfig);
+    }
 
-        Session nebulaSession;
-        try {
-            nebulaSession = nebulaPool.getSession(user, password, reconnect);
-        } catch (NotValidConnectionException | IOErrorException | AuthFailedException | ClientServerIncompatibleException e) {
-            throw new SQLException(e);
-        }
-        return nebulaSession;
+    @Override
+    public int getMajorVersion() {
+        return 1;
+    }
 
+    @Override
+    public int getMinorVersion() {
+        return 0;
     }
 
-    public void closePool() {
-        nebulaPoolConfig = null;
-        nebulaPool.close();
-        nebulaPool = null;
-        log.info("NebulaDriver closed");
+    @Override
+    public boolean jdbcCompliant() {
+        return false;
     }
 
     /**
      * connect to the NebulaGraph
      */
     @Override
-    public Connection connect(String url, Properties connectionConfig) throws SQLException {
+    public Connection connect(String url, Properties properties) throws SQLException {
         if (!acceptsURL(url)) {
             throw new SQLException("url: " + url + " is not accepted, " +
                     "url example: jdbc:nebula://graphSpace " +
```

Comment:
url example can be changed to jdbc:nebula://host_ip:port,host_ip2:port...//graphSpace

Reply:
Right, good catch